### ISSUE TITLE
Plains of Pain template help?

### ISSUE THREAD

**Issue by AnThRaXSOI** [CubeCoders/AMPTemplates#850]
Hey everyone! I’m a complete noob with all of this stuff, I was attempting to create a template for Plains of Pain so I could setup a server for myself and some friends. Any chance I can get someone’s assistance with doing this? I’ve been googling and messing around with no success. It would be GREATLY appreciated! 

Thanks in advance and keep doing what you guys are doing because this software is AMAZING!!

**Comment by Greelan** [comment]
Doesn't look like they have made the dedicated server available for self hosting. You need to rent from 4NetPlayers

**Comment by AnThRaXSOI** [comment]
> Doesn't look like they have made the dedicated server available for self hosting. You need to rent from 4NetPlayers

Seems like they've changed some things and we are now able to create our own hosted worlds. Still not sure how to go about setting up a template for AMP though. :-(

**Comment by Greelan** [comment]
I don't see any evidence that the dedicated server is broadly available. In-game multiplayer is not the same as running a dedicated server

**Comment by AnThRaXSOI** [comment]
[Plains_of_Pain_-_Dedicated_Server_-_Windows_Setup.pdf](https://github.com/user-attachments/files/15980276/Plains_of_Pain_-_Dedicated_Server_-_Windows_Setup.pdf)

**Comment by AnThRaXSOI** [comment]
> I don't see any evidence that the dedicated server is broadly available. In-game multiplayer is not the same as running a dedicated server

I got this directly from the Dev of the game.
[Plains_of_Pain_-_Dedicated_Server_-_Windows_Setup.pdf](https://github.com/user-attachments/files/15980281/Plains_of_Pain_-_Dedicated_Server_-_Windows_Setup.pdf)

**Comment by Greelan** [comment]
Great, that will make it easy then.

**Comment by AnThRaXSOI** [comment]
> Great, that will make it easy then.

I guess I'm too dumb. LOL

**Comment by AnThRaXSOI** [comment]
Was there any progress with this Greelan? I’ve not been successful, but I’ve no where near your knowledge with this stuff.

**Comment by Greelan** [comment]
Yeah, it was done last week. But the server is badly built. It basically limits you to only using the default game port, and using the command line arg to set that port stops the query port from being opened. I've noted this to the dev

**Comment by AnThRaXSOI** [comment]
Ah okay, is it anywhere I can get it? I didn’t see it in your GitHub, that’s why I was asking if there was any progress. I appreciate the time you’ve taken for this!

**Comment by Greelan** [comment]
`Greelan/AMPTemplates:dev`

But you won't be able to connect to the server until the devs fix their issue.

**Comment by IceOfWraith** [comment]
This was previously added.
